src/activities/components/ActivityList.vue

```diff
@@ -426,6 +430,40 @@ export default {
       this.slots = 'all'
       this.type = 'all'
     },
+    checkArchivedTypes () {
+      const archivedTypes = {}
+
+      for (const activityType in this.activityListTypeFilter) {
+        // count the number of archived types and store in a hashmap, defaulting to zero
+        if (this.activityListTypeFilter[activityType].status === 'archived') {
+          archivedTypes[this.activityListTypeFilter[activityType].id] = 0
+        }
+      }
+
+      console.log(this.activities)
+
+      for (const num in this.activities) {
+        // count the number of activities for each archived type and update the hashmap
+        const activity = this.activities[num]
+        if (activity.activityType.id in archivedTypes) {
+          const id = activity.activityType.id
+          archivedTypes[id] += 1
```

Comment:
The actual count doesn't really matter here, the only important thing is if there are none, or some.